src/components/Logo.tsx

```diff
@@ -0,0 +1,78 @@
+import { Input } from "design-react-kit";
+import React, { ChangeEvent } from "react";
+
+import { WithNamespaces, withNamespaces } from "react-i18next";
+
+import "./Logo.css";
+
+type OwnProps = {
+  errorLogoUpload: boolean;
+  isSubmitEnabled: boolean;
+  isValid: boolean;
+  logoPath: string;
+  logoUploaded: boolean;
+  onChangeHandler: (event: ChangeEvent<HTMLInputElement>) => void;
+  onError: () => void;
+  onSubmitHandler: () => void;
+  nameInput: string;
+  nameButton: string;
+};
+
+type Props = WithNamespaces & OwnProps;
+
+const Logo = ({
```

Comment:
Since this component wraps an image's upload, maybe we can name it `UploadLogo`.